.env

```diff
@@ -1,5 +1,5 @@
 APP_IMAGE=gdcc/dataverse:unstable
-POSTGRES_VERSION=16
+POSTGRES_VERSION=17
```

Comment:
docker/compose/demo/compose.yml still has "image: postgres:13"

I assume we should bump that too.

Reply:
whoops! good catch.